### PR TITLE
Marian's pull request for new Tesla class

### PR DESCRIPTION
**Pull request by marianmwang**
created new Tesla class that implements tradable and drivable
included test file
changed trader to initialize some teslas

### CODE REVIEW

File: src/main/java/Tesla.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    }
+
+    @Override
+    public void downgradeSpeed() {
+        this.maxSpeed -= 5;
+    }
+
+    @Override
+    public int getPrice() {
+        return 5;
```
Comment (Silk8Worm) [inline src/main/java/Tesla.java]: No way you get a Tesla for 5 anything.

File: src/test/java/TeslaTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/*
+ * This file contains sample JUnit test cases for Horse.java
```
Comment (Silk8Worm) [inline src/test/java/TeslaTest.java]: AHA. You forgot it too.

**Review by Silk8Worm — DISMISSED** [review]
Error's too hard to find.

File: src/main/java/Tesla.java (hunk 1 above)
Comment (marianmwang) [reply src/main/java/Tesla.java]: facts, price has been changed to 30000

File: src/test/java/TeslaTest.java (hunk 2 above)
Comment (marianmwang) [reply src/test/java/TeslaTest.java]: ... changed to Tesla.java

**Review by marianmwang — COMMENTED** [review]
reviewed comments and finished

**Comment by marianmwang** [comment]
merged Tesla class

**Pull request merged**